lib/hammer_cli_foreman_discovery/discovery.rb

```diff
@@ -83,6 +90,21 @@ def request_params
       build_options
     end
 
+    class AutoProvisionCommand < HammerCLIForeman::SingleResourceCommand
+      action :auto_provision
+      command_name "auto_provision"
```

Comment:
We use dash separation in command and parameter names in `hammer-cli-foreman`. I think it would be better to follow that convention.